packages/grid/_modules_/grid/components/toolbar/GridToolbarExport.tsx

```diff
@@ -7,28 +7,44 @@ import MenuItem from '@material-ui/core/MenuItem';
 import { isHideMenuKey, isTabKey } from '../../utils/keyboardUtils';
 import { GridApiContext } from '../GridApiContext';
 import { GridMenu } from '../menu/GridMenu';
-import { GridExportOption } from '../../models';
+import { GridExportOption, GridExportFormat, GridExportConfiguration } from '../../models';
 
-export const GridToolbarExport = React.forwardRef<HTMLButtonElement, ButtonProps>(
-  function GridToolbarExport(props, ref) {
+type GridToolbarExportProps = ButtonProps & { exportConfiguration?: GridExportConfiguration };
+
+export const GridToolbarExport = React.forwardRef<HTMLButtonElement, GridToolbarExportProps>(
+  function GridToolbarExport({ exportConfiguration, ...buttonProps }, ref) {
```

Comment:
Our convention is to use `other` for the rest of the props.

```suggestion
  function GridToolbarExport({ exportConfiguration, ...other }, ref) {
```

Reply:
Change applied.